### DISCUSSION TITLE
What collaborative tools to use?

### DISCUSSION THREAD

**Discussion by salmonb** [webfx-project/webfx-website#2] [General]
This discussion is to decide what tools we should use for our collaborative work on the website.

Also have we considered all Github tools? We didn't talk about the Github project boards (I can give you access). Have you used them before? Or do you think they are too simple for our needs?

**Comment by ivan-velikanov** [comment]
Confluence is one option, there are other cheaper and free alternatives, i will get back on this.

**Reply by salmonb** [reply]
> @salmonb Can you start a google drive folder and share the link here? It can have both - a presentation and a document, either one can be used based on requirement. At the end, the document should turn into html webpage(s) - as a blog or documentation, the presentation would turn into the website front page.

Regarding your previous request Ivan, I created a google drive folder with a google slide inside (which is a draft proposition for the new website card). I'm sharing this folder with you. Pedro, you can write to me at the email address displayed on my Github profile, and I will invite you as well. Please feel free to amend the card, or duplicate it, create new files, etc...

**Reply by ivan-velikanov** [reply]
I didn't get access or invite to any folder.
As for alternative to confluence we can use https://spaceli.io - it converts google docs into confluence style knowledge base. Again it is just a substitute.

**Reply by salmonb** [reply]
I shared with the email you used when you emailed me. You didn't receive the link to that email address?

**Reply by ivan-velikanov** [reply]
Nope I didn't receive, and I sent you an email btw.
**Probably you should also create a google form** for people who are interested to participate in the discussion to drop their email id so that they can be added. Also you kind of get a database of potential users and testers. These are simple sweet solutions without any harm.

**Reply by salmonb** [reply]
But I also replied to your email, you didn't receive it?
Good idea for the google form. But I need first to sort this email issue.
